site/access-control.xml

```diff
@@ -358,16 +441,22 @@ limitations under the License.
       </p>
 
 <pre class="example">
+# rabbitmq.conf
+#
+auth_backends.1.authn = ldap
+auth_backends.1.authz = internal
+auth_backends.2       = internal
```

Comment:
Ok, so disregard my previous comment, because authentication and authorization backends are organized as pairs. Sorry for the noise!